ext/SpeedyWeatherGeoMakieExt.jl

```diff
@@ -74,7 +74,7 @@ function SpeedyWeather.animate(
 
     # Get dimensions
     lon = longitude_shift_180(ds["lon"][:]) # longitudes -180 to 180
-    lat = ds["lat"][:]
+    lat = sort(ds["lat"][:]) # lats -90 to 90
```

Comment:
if we can produce an example that `90..-90` doesn't work correctly with GeoMakie then we should flag this as an issue there ...

Reply:
Ok in fairness 90...-90 does actually work, so I guess we can leave this as-is if we want? It makes sense to me at least from a readability standpoint to force the lat/lons to ascend (then the vertical flip is better justified) but I suppose not totally necessary! 

<img width="354" height="449" alt="Screenshot 2025-12-30 at 17 07 57" src="https://github.com/user-attachments/assets/ae087026-481e-49b3-a61a-00f69774f1e1" />